src/gccrs/args.rs

```diff
@@ -1,14 +1,14 @@
 //! This module interprets arguments given to `rustc` and transforms them into valid
 //! arguments for `gccrs`.
 
-use std::path::PathBuf;
+use std::{path::PathBuf, process::Command};
 
 use getopts::{Matches, Options};
 
 use super::{Error, Result};
 
 /// Crate types supported by `gccrs`
-#[derive(Clone, Copy)]
+#[derive(Clone, Copy, PartialEq)]
```

Comment:
NIT: can also derive `Eq`, since this is also reflexive (`a == a`)

```suggestion
#[derive(Clone, Copy, PartialEq, Eq)]
```